src/utils/slugModifiers/removeLegacySlugSuffix.ts

```diff
@@ -0,0 +1,7 @@
+import isSlugLegacy from "./isSlugLegacy";
+
+const removeLegacySlugSuffix = (slug: string) => {
+  return isSlugLegacy(slug) ? slug.slice(0, -2) : slug;
```

Comment:
Again, best not to have -2 being fixed i'd say.
Either `slug.slice(0, -(LEGACY_PROGRAMME_SUFFIX.length))` (if that works)
or `const [firstPart] = slug.split(LEGACY_PROGRAMME_SUFFIX)`